React/Modules/RCTUIManager.m

```diff
@@ -224,6 +224,8 @@ @implementation RCTUIManager
   NSDictionary *_componentDataByName;
 
   NSMutableSet<id<RCTComponent>> *_bridgeTransactionListeners;
+  NSMutableSet<id<RCTUIManagerObserver>> *_uiManagerObservers;
```

Comment:
If we properly remove the observer when we dealloc the animated module it shouldn't be an issue right?